<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ripes::CacheWidget</class>
 <widget class="QWidget" name="Ripes::CacheWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
     <item>
      <widget class="CacheConfigWidget" name="cacheConfig" native="true"/>
     </item>
     <item>
      <widget class="CacheView" name="cacheView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CacheConfigWidget</class>
   <extends>QWidget</extends>
   <header>cachesim/cacheconfigwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CacheView</class>
   <extends>QGraphicsView</extends>
   <header>cachesim/cacheview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
